components/backup-stream/src/endpoint.rs

```diff
@@ -742,6 +731,21 @@ where
         metrics::update_task_status(TaskStatus::Running, &task_name);
     }
 
+    // clean the safepoint created at pause(if there is)
+    fn clean_pause_guard_id_for_task(&self, task_name: &str) {
+        self.pool.spawn(root!("unregister_task";
```

Comment:
both load/unload_task will invoke this function. so consider another proper name for it?

Reply:
`pause_task` also calls the function `unload_task`. I wonder if it should also remove the pause guard too? I think it is so that the new function `clean_pause_guard_id_for_task` can be called in `unload_task` instead of `unregister_task`.

Reply:
set gcsafepoint need to handle the error `pd_client::Error::UnsafeServiceGcSafePoint` if gc safe point exceeds the task checkpoint.

Reply:
Maybe we need to drop the gc safepoint in BR to remove the scaled-in TiKV gc safepoint.